python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -45,6 +45,8 @@ class SampledSubgraphImpl(SampledSubgraph):
     ] = None
     original_row_node_ids: Union[Dict[str, torch.Tensor], torch.Tensor] = None
     original_edge_ids: Union[Dict[str, torch.Tensor], torch.Tensor] = None
+    # Used to fetch sampled_csc.indices if it is missing.
+    _sampled_edge_ids: Union[Dict[str, torch.Tensor], torch.Tensor] = None
```

Comment:
ditto: _edge_ids_in_fused_csc_sampling_graph

The name has to be clear since it is indeed non-intuitive.

Reply:
#7704